_posts/2016-09-14-marissa-vichayapai.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "#MYAAPIVOTE is for Resisting Erasure"
+category: ourstories
+tags: 
+author: Marissa Vichayapai
+featured-image: '/static/images/featured/2016-09-14-marissa-vichayapai.jpg'
+featured-alt-text: "Selfie of an Asian woman with #MyAAPIVote is for Resisting Erasure written at the bottom"
+tweet: "I vote to resist the erasure of my family, friends...and myself. #MyAAPIVote"
```

Comment:
This tweet will be truncated. Please add the URL encoding characters before merge.